cli/args/config_file.rs

```diff
@@ -524,8 +526,13 @@ impl ConfigFile {
       if checked.insert(ancestor.to_path_buf()) {
         for config_filename in CONFIG_FILE_NAMES {
           let f = ancestor.join(config_filename);
-          match ConfigFile::read(f) {
+          match ConfigFile::read(&f) {
             Ok(cf) => {
+              info!(
+                "{} found at '{}'",
+                colors::intense_blue("Config file"),
+                f.display()
+              );
```

Comment:
Please let's not do this. Can we instead make this a `--log-level=debug`?

Reply:
Oh shoot 🤦 sorry, I missed that it's not using `log::info`

Reply:
well, its not now, before it was. should we make it print to log::info for watch flag, or just have it everywhere behind debug?